sdk/monitor/azure-monitor-opentelemetry-exporter/tests/test_base_exporter.py

```diff
@@ -1,6 +1,7 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I think there are some unit tests we can add for `test_storage.py` as well right?